### ISSUE TITLE
2.11.5 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3775]
Release project board:
https://github.com/orgs/az-digital/projects/118

- #3715 
- #3570 
- #3738
- #3722 
- #3736 
- #3749 
- #3745 
- #3748 
- #3759 
- #3724 
- #3767

## Pick list
```
git cherry-pick b9cc5ee7aa1960d931289c94c81b838cde3b98ed
git cherry-pick a2ce0e916d987b7e42e9050ffcd4c61f9a78d895
git cherry-pick a75f7944f04fc396040d4b138304a53a42e9e34b
git cherry-pick ce3ec70c637625f3d5661882bfb6f986bb17053c
git cherry-pick e5798b99d983078d7237762914b706e5e509c095
git cherry-pick f4fed8add3579b3975acb4922a04803ee24f63d3
git cherry-pick f1f878b6d5a4f973c6f23640ed15b796d5822d06
git cherry-pick 56401980c3954994313eaa606ededbbad48f1a2c
git cherry-pick 26cfead1554081f59746b7cb207004849a8f9a7e
git cherry-pick e778f2b9497d08d3cdfb3d98c694e1b19998c051
git cherry-pick 38cec4a9203d49026583fd001e3be7be81772dc7
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.11.4..HEAD --pretty='format:- %s'`
```
- Close #3765 Change top-level Quickstart toolbar link to an internal page instead of an external page. (#3767)
- #3717, az-digital/az-quickstart-pantheon#282: Add ImageMagick module. (#3724)
- Closes #3758: Ensure all external and minified library assets are properly configured. (#3759)
- Closes #3747 Restore the display of 12:00 am and 12:00 pm as "midnight" and "noon" for events (#3748)
- Closes #3744 Prevent views using Quickstart Exposed Filters from scrolling when checkboxes are checked (#3745)
- Closes #1212: Add views_bulk_operations as optional module. (#3749)
- Update drupal/blazy requirement from 3.0.10 to 3.0.11 (#3736)
- Update drupal/role_delegation requirement from 1.2.0 to 1.3.0 (#3722)
- Fixes #3737 SlevomatCodingStandards (#3738)
- Fixes #3074, Fixes #3568 Course Importer Missing Content, Course Instructor Person Links (#3570)
- Closes #3714 Extra main landmark role in page template (#3715)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.11.5
